docs/contributors/repository-management.md

```diff
@@ -150,6 +150,16 @@ If you’d like a template to follow:
 >
 > For more details, please see ____ and ____.
 
+## Teams
+
+Two GitHub teams are used in the project.
+
+* [Gutenberg Core](https://github.com/orgs/WordPress/teams/gutenberg-core): A team composed of people that are actively involved in the project: attending meeting regularly, participating in triage sessions, performing reviews regularly, working on features and bug fixes and performing plugin and npm releases.
+
+* [Gutenberg](https://github.com/orgs/WordPress/teams/gutenberg): A team composed of contributors with at least 2–3 meaningful contributions to the project.
+
+Feel free to ask to be added to the repository in the #core-editor slack channel.
```

Comment:
It's not clear to me (and presumably to the contributor) what's meant by "added to the repository" . Is that specifically a reference to the second team "Gutenberg" which is noted above as being more open after a certain number of contributions?

Also, we assume people know how to get to Slack, but it doesn't hurt to be explicit at every point it's mentioned.

```suggestion
If you meet this criteria of several meaningful contributions having been accepted into the repository and would like to be added to the Gutenberg team, feel free to ask in the [#core-editor Slack channel](https://make.wordpress.org/chat/).
```

Would one of the weekly meetings be an appropriate time to suggest? It may not matter, though from their perspective it could be more comfortable to ask at a time dedicated in part to these sorts of administrative tasks.

Reply:
I suggest that we create an issue template that people can use for being added to the Gutenberg Core team.  It can include  a request to list the pulls/contributions they made that were approved and merged and ask for a brief explanation why they want added.   We could then maybe have a label for those issues.  Then as a part of the #core-editor meeting these issues can be brought up for brief review and ask if there are any objections, if none then add them to the team.  Reasons:

1. It formalizes the process a bit more and provides a clear actionable step for people who want to be on the team to follow.
2. The issue template can gather the information needed to decide whether to add the person to the team.
3. There's historical record in the repository for when people were added to the team and what contributions they made (others thinking about how to join can review to see the kinds of contributions that were made)
4. It provides an escape hatch if someone requests to be on the team that others might have a red flag about.

Reply:
The issue template could also include a checkbox list of components (used in the `codeowners` file) of what components they'd like to be notified about too.

Reply:
> Would one of the weekly meetings be an appropriate time to suggest? 

It could add some burden as people might not be available at these times (the contributors).

> issue template

I think we disagree often in similar ways :) (very formal VS something middle ground). I don't think both approaches have values.

In this particular case, I feel an issue template is intimidating for the people that want to be added as contributors, they might feel like they need to sell themselves, where I think it's better if it's a no brainer. Just ask `Can I be added to the gutenberg team?` in a slack channel.